.travis.yml

```diff
@@ -10,15 +10,16 @@ matrix:
     include:
         - python: "2.7"
           env: TOXENV=flake8,docs
```

Comment:
I only just noticed this!  We used to have a Travis job to run flake8 and sphinx, but now that tox is no longer used, TOXENV will have no effect.